src/app/SlackBlocks/makeTaskEditBlock.ts

```
@@ -0,0 +1,139 @@
+}
+
+// task.detailをパースしてelementsを抽出する関数
+function parseTaskDetail(detail: string): any[] {
+  try {
+    return JSON.parse(detail).elements;
+  } catch {
+    return [
+      {
+        type: "rich_text_section",
+        elements: [{ type: "text", text: "情報のパースに失敗しました。開発者に連絡してください。" }],
+      },
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)